app/src/main/java/org/mozilla/fenix/perf/Performance.kt

```diff
@@ -37,4 +45,25 @@ object Performance {
             activity.reportFullyDrawn()
         }
     }
+
+    /**
+     * Processes intent for Performance testing to remove protection pop up ( but keeps the TP
```

Comment:
nit: in hindsight, explaining this comment at a higher level in code instead would make this is clearer and less likely to get out of date:
```kotlin
fun processIntentIfPerformanceTest(intent: Intent, context: Context) {
    if (!isPerformanceTest()) { return }

    disableOnboarding()
    disableTrackingProtectionPopups()
}